command/task_create.go

```diff
@@ -274,9 +274,18 @@ func handleDeprecations(ui mcli.Ui, tc *config.TaskConfig) error {
 				action := generateServiceModuleInputBlockAction(tc.Services)
 				ui.Output(generateServiceFieldMsg(action))
 			case *config.ScheduleConditionConfig:
-				action := generateServicesAction(tc.Services)
+				action := generateServiceModuleInputBlockAction(tc.Services)
 				ui.Output(generateServiceFieldMsg(action))
 			}
+		} else if tc.ModuleInputs != nil {
```

Comment:
Would we want to make this a separate if-statement rather than an else-if statement? That would allow returning messages for module input when a user configures a task with `condition "services"` and `module_input "services"`. (It would impact the final else-statement)